### ISSUE TITLE
More detailed build instructions.

### ISSUE THREAD

**Issue by felipesanches** [daltonmaag/scope-one#14]
I have checked out this project repository and I am trying to follow the build instructions.

I have created the virtual-env, as instructed, but I got a python error when running build.sh:

```sh
(scope_build_env) felipe@guarana:~/devel/github_felipesanches/scope-one$ ./build.sh 
Traceback (most recent call last):
  File "tools/build.py", line 14, in <module>
    from ufo2fdk import OTFCompiler
ImportError: No module named 'ufo2fdk'
```

Since I have both python 2 and python 3 in my system, I think this may be causing trouble. I was told that the virtual-env would take care of that. But it does not seem to be the case here. So I used 'pip3 install -r requirements' and also changed python to python3.4 in the build.sh script. Now it actually runs the build script, but fails for another reason:

ttfautohint: unrecognized option '-D'

I am using ttfautohint installed from apt-get. Maybe I should install a different version? Can you inform which version you are using that handles well this -D command line option ? This seems like important dependency info to be placed in the documentation. The -m option also seems unsupported by my stable ttfautohint binary.

**Comment by anthrotype** [comment]
Hi Felipe,
Could you please paste the commands you entered to set up the environment? What OS and Python version are you on?
We used ttfautohint 1.4.1 to build the font, and we haven't tested earlier versions. I'll add a note to the readme about this.
Thanks for the report.

**Comment by davelab6** [comment]
I guess  Ttfautohint should be from git master, or at least latest release.

**Comment by felipesanches** [comment]
OK. I just built ttfautohint from git, installed it and got the build of this font project working :-)

**Comment by anthrotype** [comment]
excellent! :+1:

**Comment by felipesanches** [comment]
Feel free to close this issue is you have already improved the documentation.

**Comment by anthrotype** [comment]
Closed #14 via https://github.com/daltonmaag/scope-one/commit/25cf0810b070d56ff2e105ed20f8f03badbbb565
